css/style.css

```diff
@@ -18,11 +19,86 @@ body {
   border: none;
 }
 
+.result {
+  display: none;
+}
+
 .result .description {
   font-size: 22px;
   line-height: normal;
 }
 
+.flex-container {
+  display: flex;
+}
+
 #location {
+  margin-top: 1em;
+  /* font-weight: bold; */
+  /* font-size: large; */
+}
+
+.current,
+.forecast {
+  margin-top: 2em;
+  margin: auto;
   display: none;
 }
+
+.current {
+  width: 40%;
+}
+
+.forecast {
+  width: 80%;
+  justify-content: space-between;
+}
+
+#button-container {
+  margin-top: 1em;
+}
+
+/* .toggle-buttons {
```

Comment:
Can these al be removed? If you plan to use them in the future, add a comment, otherwise delete them as you can always get them later by looking back in previous commits.

Reply:
Yes, they have been removed in the most recent push